cli/command/stack/kubernetes/cli.go

```diff
@@ -58,7 +60,10 @@ func WrapCli(dockerCli command.Cli, opts Options) (*KubeCli, error) {
 	cli.kubeNamespace = opts.Namespace
 	if opts.Namespace == "" {
 		configNamespace, _, err := clientConfig.Namespace()
-		if err != nil {
+		switch {
+		case os.IsNotExist(err), os.IsPermission(err):
+			return nil, errors.Wrap(err, "unable to load Kubernetes configuration")
```

Comment:
As @thaJeztah suggested, I think we can just say `unable to load configuration`, kubernetes is already present elsewhere :stuck_out_tongue: